project_milestone/README.rst

```diff
@@ -0,0 +1,92 @@
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproject-lightgray.png?logo=github
+    :target: https://github.com/OCA/project/tree/12.0/project_milestone
+    :alt: OCA/project
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/project-12-0/project-12-0-project_milestone
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/140/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module provides a project milestones on projects.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Have Manager rights for Project group to edit projects.
+#. On the project form, set the "Use Milestones" option to enable milestones for the project.
+#. Add the milestones to the project via the milestone tab on the project form.
+#. Tasks will show a "Milestone" where you can set tasks into the proper milestone.
+#. You also have the group by "Milestones" for task views.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/project/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/project/issues/new?body=module:%20project_milestone%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* Patrick Wilson
+
+Contributors
+~~~~~~~~~~~~
+
+* Patrick Wilson <patrickraymondwilson@gmail.com>
+
+Maintainers
+~~~~~~~~~~~
+
+This module is maintained by the OCA.
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+.. |maintainer-patrickrwilson| image:: https://github.com/patrickrwilson.png?size=40px
+    :target: https://github.com/patrickrwilson
+    :alt: patrickrwilson
+
+Current `maintainer <https://odoo-community.org/page/maintainer-role>`__:
+
+|maintainer-patrickrwilson| 
+
+This module is part of the `OCA/project <https://github.com/OCA/project/tree/12.0/project_milestone>`_ project on GitHub.
```

Comment:
I assume that all the references to 12.0 will be updated when the readme gets processed?